icu4c/source/test/intltest/dtifmtts.cpp

```diff
@@ -1620,4 +1621,67 @@ void DateIntervalFormatTest::testTicket12065() {
 }
 
 
+void DateIntervalFormatTest::testFormattedDateInterval() {
+    IcuTestErrorCode status(*this, "testFormattedDateInterval");
+    LocalPointer<DateIntervalFormat> fmt(DateIntervalFormat::createInstance(u"dMMMMy", status), status);
```

Comment:
I should pass the locale into the constructor, or else this test might not pass on foreign machines.